scss/_custom-forms.scss

```diff
@@ -220,11 +223,7 @@
   background: $custom-select-background;
   background-color: $custom-select-bg;
   border: $custom-select-border-width solid $custom-select-border-color;
-  @if $enable-rounded {
-    border-radius: $custom-select-border-radius;
-  } @else {
-    border-radius: 0;
-  }
+  @include border-radius($custom-select-border-radius, 0);
```

Comment:
I didn't really like the `if/else` statement here, so I tweaked the `border-radius` mixin a bit to simplify this code here.